### DISCUSSION TITLE
POJ 3219 二项式系数

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#368] [Announcements]
# POJ 3219 二项式系数

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/2014/08/18/POJ-3219-Binomial-Coefficient/

**Comment by drmingdrmer** [comment]
666...

a题专用文件头

**Reply by Xuanwo** [reply]
2014 年的黑历史都被翻出来了

**Reply by drmingdrmer** [reply]
看到了有趣的题目就点了...
